docling_core/types/doc/base.py

```diff
@@ -59,27 +59,72 @@ def scaled(self, scale: float) -> "BoundingBox":
         :param scale: float:
 
         """
-        out_bbox = copy.deepcopy(self)
-        out_bbox.l *= scale
-        out_bbox.r *= scale
-        out_bbox.t *= scale
-        out_bbox.b *= scale
-
-        return out_bbox
+        warnings.warn(
```

Comment:
I see absolutely no reason to deprecate methods, they can be used exactly as they were. If you want to support separate x and y scales, then the argument scale can be a Union[float, Size] and the implementation has to check.

Reply:
done